app/[locale]/blog/[slug]/page.tsx

```
@@ -419,9 +418,9 @@ const BlogPostContent = ({
 
               <div className="blog-content prose prose-lg max-w-none">
                 <AlertIconInjector contentKey={post.slug}>
-                  {/* Apply runtime sanitization via SafeHtml for defense-in-depth security */}
-                  <SafeHtml
-                    html={contentWithIds}
+                  {/* Note: contentWithIds is sanitized at build time; runtime sanitization not required */}
+                  <div
+                    dangerouslySetInnerHTML={{ __html: contentWithIds }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)